raster/src/main/scala/geotrellis/raster/stitch/Stitcher.scala

```diff
@@ -33,3 +33,37 @@ object Stitcher {
   }
 
 }
+
+
+trait StitcherR[T <: CellGrid, R <: CellGrid] extends Serializable {
```

Comment:
Do we need to add StitcherR now, or can we just type the test code correctly to avoid having the duplicate classes and keep with Stitcher?